sonic-xcvrd/xcvrd/xcvrd.py

```diff
@@ -619,6 +631,9 @@ def post_port_dom_info_to_db(logical_port_name, port_mapping, table, stop_event=
         if not _wrapper_get_presence(physical_port):
             continue
 
+        if _wrapper_is_flat_memory(physical_port) == True:
```

Comment:
So, no DOM (TRANSCEIVER_DOM_SENSOR) STATE_DB info is populated for xcvrs with flat memory?